src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,106 @@
+package duke.ui;
+
+import duke.DukeException;
+import duke.todo.Task;
+
+public class Ui {
+    StringBuilder messageBuffer;
```

Comment:
Never thought of using this to arrange the string fully before printing or returning it. Thanks!

Reply:
It could be simple concatenation because of the lightweight nature of this project. However, I do feel like doing StringBuilder.